source/guide_focalboard.rst

```diff
@@ -0,0 +1,98 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: web
+.. tag:: project-management
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/focalboard.svg
+      :align: center
+
+##########
+Focalboard
+##########
+
+.. tag_list::
+
+Focalboard_ is an open source, self-hosted alternative to Trello, Notion, and Asana. 
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
```

Comment:
The [official Installation guide](https://www.focalboard.com/download/personal-edition/ubuntu/#install-postgresql-recommended) recommends the usage of PostgreSQL. So I’d suggest to add a section that describes this:

```
It’s recommended to use a PostgreSQL database. Install and configure PostgreSQL server as described in the :lab:`PostgreSQL guide <guide_postgresql>`.
```